modules/secure-cloud-run/variables.tf

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+variable "location" {
+  description = "The location where resources are going to be deployed."
+  type        = string
+}
+
+variable "region" {
+  description = "Location for load balancer and Cloud Run resources."
+  type        = string
+}
+
+variable "serverless_project_id" {
+  description = "The project to deploy the cloud run service."
+  type        = string
+}
+
+variable "vpc_project_id" {
+  description = "The host project for the shared vpc."
+  type        = string
+}
+
+variable "key_name" {
```

Comment:
Let's provide defaults for all of the resource which we do not need explicitly need from the user.